apps/dashboard/src/components/show.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactNode } from 'react';
+
+export const Show = ({ when, fallback, children }: { when: boolean; fallback?: ReactNode; children: ReactNode }) => {
```

Comment:
Nit picky comment: I always prefer ternary JSX instead of this component. There is always an extra import without a significant DX benefit.

Reply:
yeah, we agreed to proceed with ternary as it's better also working with TS